source/usage-examples/struct-tagging.txt

```diff
@@ -0,0 +1,69 @@
+===========================
+Set Custom BSON Field Names
+===========================
+
+.. default-domain:: mongodb
+
+You can customize the way that the Go Driver converts Go
+structs to :manual:`BSON </reference/bson-types/>` using 
+**struct tags**, which are optional instructions you can use with struct
+fields.
+
+The following code declares a ``BlogPost`` struct, where the ``WordCount`` field
+has a struct tag that sets a custom BSON field name of ``word_count``.
+The driver marshals the other fields as the lowercase of the struct
+field name by default:
+
+.. literalinclude:: /includes/usage-examples/code-snippets/struct-tag.go
+   :start-after: begin struct
+   :end-before: end struct
+   :language: go
+   :dedent:
+
+The following code creates a ``BlogPost`` instance and inserts it
+into the ``posts`` collection of the ``sample_training`` database,
+converting the struct to BSON in the process:
```

Comment:
Suggestion: I think this line is a critical action that should get its own sentence telling the user how the tag you created right before reflects here.